release-schema.json

```
@@ -460,7 +460,18 @@
         "id": {
           "title": "ID",
           "description": "A local identifier for this bid",
-          "type": "string"
+          "type": "string",
+          "minLength": 1
+        },
+        "identifiers": {
+          "title": "Identifiers",
+          "description": "Identifiers for this bid. This field can be used to provide internal identifiers for the bid, such as identifiers from a buyer's document management system or procurement system.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SimpleIdentifier"
+          },
+          "wholeListMerge": true,
+          "uniqueItems": true
```

We can keep the strict validation on the new field.

```suggestion
          "uniqueItems": true,
          "minItems": 1
```